lua/neo-tree/sources/common/components.lua

```diff
@@ -275,7 +275,11 @@ M.icon = function(config, node, state)
     elseif node:is_expanded() then
       icon = config.folder_open or "-"
```

Comment:
I realize this problem pre-existed, but now I see that it doesn't make sense that there is only a symlink icon for closed dirs and not open ones. This should be fixed.